src/Core/src/Platform/Android/MauiDatePicker.cs

```diff
@@ -45,7 +45,8 @@ void Initialize()
 				DrawableCompat.Wrap(Background);
 
 			Focusable = true;
-			FocusableInTouchMode = false;
+			FocusableInTouchMode = true;
```

Comment:
I don't think we can toggle this to true. @hartez specifically set this to false [here](https://github.com/dotnet/maui/commit/7604a5dcfc0838726cc27533d5da83688d30ad32#diff-10e0d2d766061f4ad66267fce77f9973e62050f7ca9a66c7ae626fd6a4162d54R47) so that the picker wouldn't open up on focus.

What do you think about removing all the android parts of this PR and creating new APIs for "opened/closed"?